contracts/precrime/IPreCrime.sol

```diff
@@ -0,0 +1,47 @@
+// SPDX-License-Identifier: BUSL-1.1
```

Comment:
export to and import from @layerzerolabs/lz-evm-sdk-v1

Reply:
@layerzerolabs/lz-evm-sdk-v1 doesn't have Precrime

Reply:
need to export and publish it, it's here https://github.com/LayerZero-Labs/monorepo/tree/main/packages/layerzero-v1/evm/sdk/artifacts/contracts/precrime

Reply:
published https://www.npmjs.com/package/@layerzerolabs/lz-evm-sdk-v1?activeTab=code

Reply:
The package doesn't contain contracts source files, only artifacts, and importing Precrime doesn't work
`import "@layerzerolabs/lz-evm-sdk-v1/artifacts/contracts/precrime/PreCrimeView.sol";`